pkg/tools/backplanecli/backplanecli.go

```diff
@@ -36,42 +35,17 @@ func (t *Tool) Install() error {
 		return err
 	}
 
-	// Determine which assets to download
-	var checksumAsset *gogithub.ReleaseAsset
-	var backplaneArchiveAsset *gogithub.ReleaseAsset
-	arch := runtime.GOARCH
-	if arch == "amd64" {
-		arch = "x86_64"
+	matches := github.FindAssetsForArchAndOS(release.Assets)
+	if len(matches) != 1 {
+		return fmt.Errorf("unexpected number of assets found matching system spec: expected 1, got %d.\nMatching assets: %v", len(matches), matches)
```

Comment:
This error ends up being reused a lot in each tool's file.  I don't quite know how to suggest changing it, but storing it as a variable that could be reused each time?  Maybe not worth it, I dunno.